internal/sbi/producer/nf_discovery.go

```diff
@@ -38,10 +38,44 @@ func HandleNFDiscoveryRequest(request *httpwrapper.Request) *httpwrapper.Respons
 	return httpwrapper.NewResponse(http.StatusForbidden, nil, problemDetails)
 }
 
+func ValidateQueryParameters(queryParameters url.Values) bool {
+	NFs := [...]string{"NEF", "BSF", "SMF", "UPF", "PCF", "AMF", "CHF", "AUSF", "UDM", "UDR"}
```

Comment:
Hi @danielh1204 
Please help to check all of NFs listed below have be presented in your change:
```
const (
	NfType_NRF      NfType = "NRF"
	NfType_UDM      NfType = "UDM"
	NfType_AMF      NfType = "AMF"
	NfType_SMF      NfType = "SMF"
	NfType_AUSF     NfType = "AUSF"
	NfType_NEF      NfType = "NEF"
	NfType_PCF      NfType = "PCF"
	NfType_SMSF     NfType = "SMSF"
	NfType_NSSF     NfType = "NSSF"
	NfType_UDR      NfType = "UDR"
	NfType_LMF      NfType = "LMF"
	NfType_GMLC     NfType = "GMLC"
	NfType__5_G_EIR NfType = "5G_EIR"
	NfType_SEPP     NfType = "SEPP"
	NfType_UPF      NfType = "UPF"
	NfType_N3_IWF   NfType = "N3IWF"
	NfType_AF       NfType = "AF"
	NfType_UDSF     NfType = "UDSF"
	NfType_BSF      NfType = "BSF"
	NfType_CHF      NfType = "CHF"
	NfType_NWDAF    NfType = "NWDAF"
)
```